seed/challenges/01-front-end-development-certification/jquery.json

```diff
@@ -556,7 +556,7 @@
         "fccss",
         "  $(document).ready(function() {",
         "    $(\"#target1\").css(\"color\", \"red\");",
-        "",
+        "    ",
```

Comment:
From a user perspective, it lines up the line of code you're about to write with the previous line. So it is not really *necessary* per say, but I think it helps on the user end. To me, it's kinda like if your text editor or word processor keeps the indent of your text after you make a new line.